content/en/articles/js/es2015/classes.md

```diff
@@ -57,19 +57,19 @@ User.prototype.sayName = function() {
   return this.firstname + " " + this.lastname
 }
 
-// instanciation
+// instantiation
 var user = new User("John", "Doe")
 
 // call of the method sayName()
 console.log(user.sayName()) // John Doe
 ```
 
-## Classes expressions
```

Comment:
The snippet below is not a class expression but a class declaration. Changing the title _Syntax_ would better picture this section idea.